src/index.html

```diff
@@ -13,10 +13,16 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The `href` attribute is linking to a SCSS file (`main.scss`). HTML cannot directly use SCSS files; they need to be compiled into CSS first. Ensure that `main.scss` is compiled into a CSS file, and link to that CSS file instead.